server/app-engine/gtfsrt.py

```diff
@@ -245,7 +270,7 @@ def get_alert_feed(datastore_client, agency):
         count = 1
 
         for item in results:
-            if alert_is_current(item):
+            if alert_is_current(item, include_future_alerts):
                 print('-- ' + str(item))
```

Comment:
No one looking at the logs will know what this is about.

Reply:
I'm not sure what you mean by the logs, but I'm adding a note to service_alerts() in main.py to explain what this field is for. Does that address what you're referring to?